### ISSUE TITLE
Wrong URIs used for license concepts

### ISSUE THREAD

**Issue by acka47** [hbz/oerworldmap#1376]
See e.g. https://oerworldmap.org/resource/urn:uuid:29056e88-727b-407e-b00a-16df279ac506.json where we point to https://github.com/hbz/oerworldmap/public/json/licenses.json instead of https://oerworldmap.org/assets/json/licenses.json.

**Comment by literarymachine** [comment]
Steps to fix this:

-
Update https://github.com/hbz/oerworldmap/blob/master/public/json/licenses.json with correct URIs
- Migrate existing data with SPARQL Update

**Comment by literarymachine** [comment]
> Migrate existing data with SPARQL Update

I couldn't figure out how to do it in one go, so [here](https://beta.oerworldmap.org/sparql/update?delete=%3Fs+%3Fp+%3Fo+.&insert=%3FnewURI+%3Fp+%3Fo+.&where=%3Fs+%3Fp+%3Fo+.+FILTER+%28STRSTARTS%28STR%28%3Fs%29%2C+%22https%3A%2F%2Fgithub.com%2Fhbz%2Foerworldmap%2Fpublic%22%29%29+BIND+%28URI%28CONCAT+%28%22https%3A%2F%2Foerworldmap.org%2Fassets%22%2C+STRAFTER+%28STR%28%3Fs%29%2C%22https%3A%2F%2Fgithub.com%2Fhbz%2Foerworldmap%2Fpublic%22%29%29%29+AS+%3FnewURI%29) is the migration for all occurrences as a subject and [here](https://beta.oerworldmap.org/sparql/update?delete=%3Fs+%3Fp+%3Fo+.&insert=%3Fs+%3Fp+%3FnewURI+.&where=%3Fs+%3Fp+%3Fo+.+FILTER+%28STRSTARTS%28STR%28%3Fo%29%2C+%22https%3A%2F%2Fgithub.com%2Fhbz%2Foerworldmap%2Fpublic%22%29%29+BIND+%28URI%28CONCAT+%28%22https%3A%2F%2Foerworldmap.org%2Fassets%22%2C+STRAFTER+%28STR%28%3Fo%29%2C%22https%3A%2F%2Fgithub.com%2Fhbz%2Foerworldmap%2Fpublic%22%29%29%29+AS+%3FnewURI%29) for all as an object.

**Comment by acka47** [comment]
In the second DELETE/INSERT query I find some lines where the old URIs are still used in subject position, e.g.

```
+ <https://github.com/hbz/oerworldmap/public/json/licenses.json#other-pd> <http://www.w3.org/2004/02/skos/core#topConceptOf> <https://oerworldmap.org/assets/json/licenses.json> .
+ <https://github.com/hbz/oerworldmap/public/json/licenses.json#other-pd> <http://www.w3.org/2004/02/skos/core#inScheme> <https://oerworldmap.org/assets/json/licenses.json> .
+ <https://github.com/hbz/oerworldmap/public/json/licenses.json#other-nc> <http://www.w3.org/2004/02/skos/core#topConceptOf> <https://oerworldmap.org/assets/json/licenses.json> .
+ <https://github.com/hbz/oerworldmap/public/json/licenses.json#other-nc> <http://www.w3.org/2004/02/skos/core#inScheme> <https://oerworldmap.org/assets/json/licenses.json> .
```

**Comment by literarymachine** [comment]
> In the second DELETE/INSERT query I find some lines where the old URIs are still used in subject position, e.g.

Yeah, this only works if you actually carry out the migrations, and do so sequentially. But I managed to figure out how to [migrate subjects and objects simultaneously](https://beta.oerworldmap.org/sparql/query?q=SELECT+%3FnewS+%3Fp+%3FnewO+WHERE+%7B%0D%0A%3Fs+%3Fp+%3Fo+.+FILTER+%28STRSTARTS%28STR%28%3Fs%29%2C+%22https%3A%2F%2Fgithub.com%2Fhbz%2Foerworldmap%2Fpublic%22%29+%7C%7C+STRSTARTS%28STR%28%3Fo%29%2C+%22https%3A%2F%2Fgithub.com%2Fhbz%2Foerworldmap%2Fpublic%22%29%29%0D%0ABIND+%28IF%28STRSTARTS%28STR%28%3Fs%29%2C+%22https%3A%2F%2Fgithub.com%2Fhbz%2Foerworldmap%2Fpublic%22%29%2C+URI%28CONCAT+%28%22https%3A%2F%2Foerworldmap.org%2Fassets%22%2C+STRAFTER+%28STR%28%3Fs%29%2C%22https%3A%2F%2Fgithub.com%2Fhbz%2Foerworldmap%2Fpublic%22%29%29%29%2C+%3Fs%29+AS+%3FnewS%29%0D%0ABIND+%28IF%28STRSTARTS%28STR%28%3Fo%29%2C+%22https%3A%2F%2Fgithub.com%2Fhbz%2Foerworldmap%2Fpublic%22%29%2C+URI%28CONCAT+%28%22https%3A%2F%2Foerworldmap.org%2Fassets%22%2C+STRAFTER+%28STR%28%3Fo%29%2C%22https%3A%2F%2Fgithub.com%2Fhbz%2Foerworldmap%2Fpublic%22%29%29%29%2C+%3Fo%29+AS+%3FnewO%29%0D%0A%7D).

**Comment by literarymachine** [comment]
Fixed in #1377.
